src/modules/common/corgi/Activity/Activity.jsx

```diff
@@ -15,15 +15,18 @@ const ActivityPropTypes = {
   sender: CorgiType.sender,
 };
 
+const activityTimeOptions = { largest: 2, units: ['d', 'h', 'm', 's'] };
```

Comment:
units can be replaced with ['d', 'h'], because 'm' and 's' options are redundant

Reply:
done